docs/contributing.md

```diff
@@ -6,24 +6,56 @@ The following are guidelines for contributing to this project.
+Please refer to installation and build instructions in the [install
+documentation](install.md).
 
 ## How to Report Bugs
 
-Please open [a new issue in the appropriate GitHub repository](https://github.com/mozilla-lockbox/lockbox-extension/issues/new) with steps to reproduce the problem you're experiencing.
+Please open [a new issue in the appropriate GitHub repository][new-issue] with
+steps to reproduce the problem you're experiencing.
 
-Be sure to include as much information including screenshots, text output, and both your expected and actual results.
+Be sure to include as much information including screenshots, text output, and
+both your expected and actual results.
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository](https://github.com/orgs/mozilla-lockbox/) and search [the repository's GitHub issues](https://github.com/mozilla-lockbox/lockbox-extension/issues) to make sure your idea has not been (or is not still) considered. It may also be easier to see and search across all projects combined on our [Waffle.io planning board](https://waffle.io/mozilla-lockbox/lockbox-extension).
+First, please refer to the applicable [GitHub repository][github-repos] and
+search [the repository's GitHub issues][issues-list] to make sure your idea has
+not been (or is not still) considered. It may also be easier to see and search
+across all projects combined on our [Waffle.io planning board][waffle].
 
-Then, please [create a new issue in the GitHub repository](https://github.com/mozilla-lockbox/lockbox-extension/issues/new) describing your enhancement.
+Then, please [create a new issue in the GitHub repository][new-issue] describing
+your enhancement.
 
-Be sure to include as much detail as possible including step-by-step descriptions, specific examples, screenshots or mockups, and reasoning for why the enhancement might be worthwhile.
+Be sure to include as much detail as possible including step-by-step
+descriptions, specific examples, screenshots or mockups, and reasoning for why
+the enhancement might be worthwhile.
 
-Please keep in mind, by opening an issue we provide no guarantee the enhancement will be implemented.
+Please keep in mind, by opening an issue we provide no guarantee the enhancement
+will be implemented.
+
+## How to Contribute Code
+
+Before you get started writing code, be sure what you plan to work on is
+something we'll be able to accept. The easiest way is to look through out list
+of [good first issues][good-first-issues] and find something that sounds
+interesting. If there's something else you'd like to work on, just add a comment
+in the relevant issue and we'll be happy to discuss your plans.
+
+Once you have a patch ready, submit a pull request and request a review from
+**@mozilla-lockbox/engineering**. From there, we'll guide you through the review
+process to the eventual landing of your code!
+
```

Comment:
As agreed in a team discussion on 2018-01-26, there should be guidance on when an external contributor can expect to hear something from this team.  Something along the lines of the following (@sandysage @kfontecilla can correct me if mistaken):

```
Someone from this team will make the best effort to review or comment on your changes within two working days.
```

Reply:
Do we want to make a public commitment to that? I thought the plan was that the 2-day deadline was just for us to know about. I'm just hesitant to say we'll do this when it might be hard to guarantee for outside contributions (e.g. if you and I are both out for a couple days); for internal stuff, we'd know who's out, but for external patches, the contributor probably wouldn't.

That said, I don't have a strong feeling on the matter, so if everyone else thinks we should say this in the docs, I'm ok with it.

Reply:
The plan we discussed was to add it to public documentation, and my recollection is we had consens on "best effort" (see my sample above).  Best effort is not a guarantee.